package.json

```diff
@@ -53,6 +53,7 @@
     "progress": "^2.0.1",
     "proxy-from-env": "^1.0.0",
     "rimraf": "^3.0.2",
+    "source-map-support": "^0.5.19",
```

Comment:
Since Puppeteer users don’t need this dependency, can we move this to `devDependencies` instead?

Reply:
I thought I did this but npm disagrees. Who's to say who is right and who is wrong...